### PR TITLE
Consistent value refs

### PR DESCRIPTION
**Pull request by iwootten**
### What is the context of this PR?

Update validation references to be consistent as per the definition found here:https://github.com/ONSdigital/eq-questionnaire-validator/blob/master/doc/decisions/0005-use-consistent-value-references-in-validation.md

### How to review 

Ensure the unit tests pass.

### CODE REVIEW

**Comment by iwootten** [comment]
[Implement consistent value references in validation (L)](https://trello.com/c/5G5R7Id0/3672-implement-consistent-value-references-in-validation-l)

File: scripts/run_validator.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-branch=latest
+branch=consistent-value-refs
```
Comment (iwootten) [inline scripts/run_validator.sh]: Reminder to update this branch once https://github.com/ONSdigital/eq-questionnaire-validator/pull/14 is merged

**Comment by iwootten** [comment]
> * Min and max features still working as expected. 👍
> * You could up the version of eq-questionnaire-schemas in this PR also?. They are dependant.
> * Actions failing on validator.

It's only failing because you touched it!

**Comment by iwootten** [comment]
> * You could up the version of eq-questionnaire-schemas in this PR also?. They are dependant.

Will do once it's been merged

File: test_schemas/en/ecommerce.json [hunk 2]
```diff
@@ -253,11 +253,11 @@
                                         "id": "answerc5d3639c-dc1d-4ccc-a386-2df94b367dc3",
```
Comment (ajmaddaford) [inline test_schemas/en/ecommerce.json]: We should remove this schema.

**Pull request merged**